### PR TITLE
July migration sprint staging

### PR DESCRIPTION
**Pull request by wwelling**

### CODE REVIEW

File: bibs/referenceData/0_statisticalCodeTypes.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "name": "custodian",
+      "source": "voyager"
+    },
+    {
+      "id": "e7dff618-aad7-4af9-8d8d-a5a945b12431",
+      "name": "vger_item_status",
+      "source": "voyager"
+    }
+  ]
+}
```
Comment (jeremythuff) [inline bibs/referenceData/0_statisticalCodeTypes.json:20]: EOF new line needed

File: bibs/referenceData/identifierTypes.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "path": "identifier-types",
+  "data": [
+    {
+      "id" : "76b5963b-ccce-4715-b453-1a7354f4f597",
+      "name" : "former_bib_id",
+      "source" : "local"
+    }
+  ]
+}
```
Comment (jeremythuff) [inline bibs/referenceData/identifierTypes.json:10]: EOF new line needed

File: holdings/referenceData/holdingsNoteTypes.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      "name": "Access restrictions",
+      "source": "local"
+    },
+    {
+      "id": "7ca7dc63-c053-4aec-8272-c03aeda4840c",
+      "name": "latest_in",
+      "source": "local"
+    }
+  ]
+}
```
Comment (jeremythuff) [inline holdings/referenceData/holdingsNoteTypes.json:15]: EOF new line needed

File: holdings/referenceData/holdingsTypes.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "path": "holdings-types",
+  "data": [
+    {
+      "id" : "61155a36-148b-4664-bb7f-64ad708e0b32",
+      "name" : "Unknown",
+      "source" : "marc"
+    }
+  ]
+}
```
Comment (jeremythuff) [inline holdings/referenceData/holdingsTypes.json:10]: EOF new line needed

File: bibs/referenceData/0_statisticalCodeTypes.json (hunk 1 above)
Comment (wwelling) [reply bibs/referenceData/0_statisticalCodeTypes.json:20]: I wouldn't say needed. It is not required. But I will add them.

File: users/referenceData/addressTypes.json [hunk 5]
```diff
@@ -2,14 +2,19 @@
...
     },
     {
-      "id": "79d73d04-0248-11ea-8d71-362b9e155667",
       "addressType": "Temporary",
-      "desc": "Temporary address"
+      "desc": "Temporary",
+      "id": "0a9acd33-ce98-4f39-9e1b-a693b6d1adef"
     }
   ]
 }
```
Comment (jeremythuff) [inline users/referenceData/addressTypes.json:20]: EOF new line needed

File: users/referenceData/patronGroups.json [hunk 6]
```diff
@@ -2,34 +2,64 @@
...
     {
-      "id": "b61c14ac-00e4-11ea-8d71-362b9e155667",
-      "group": "ugrad",
-      "desc": "Undergraduate Student"
+      "group": "ungr",
+      "desc": "Undergrad Student",
+      "id": "666d2e72-2d05-4d6f-849d-d5234bf77686"
     }
   ]
 }
```
Comment (jeremythuff) [inline users/referenceData/patronGroups.json:65]: EOF new line needed

File: vendors/referenceData/categories.json [hunk 7]
```diff
@@ -2,24 +2,24 @@
...
+      "id": "62042758-5266-472a-a3e9-ea1ca0ccf056"
     },
     {
-      "id": "544459af-fc5e-4e64-9b40-acb84ac4d3aa",
-      "value": "RETURN"
+      "value": "return",
+      "id": "3c31275e-7ed8-4ec0-9b18-31243461f6ea"
     }
   ]
 }
```
Comment (jeremythuff) [inline vendors/referenceData/categories.json:25]: EOF new line needed

**Pull request merged**